### ISSUE TITLE
Add more fields in the model to predict salary more precisely [Based on the dataset]

### ISSUE THREAD

**Issue by yashraj-dudhe** [yashraj-dudhe/salary_prediction-#1]
The model currently predicts salary based on :
-country
-education level
-years of experience
I would like you to add more parameters to predict on [given in the dataset] like:
-Developer type
-Employment
-LanguageWorkedWith
-PlatformWorkedWith
-His undergraduate major
etc

All this is included in the database[csv] file in the repo, modify the code in the project.ipynb and update requirements file

**Comment by swarupsahu08** [comment]
Hi @yashraj-dudhe Can you assign me this issue so that I can work on this issue

**Comment by swarupsahu08** [comment]
hey @yashraj-dudhe I think there is something error I am getting while trying to install requirements.txt ,is there any thing wrong you can check or can you help me to resolve the issue

**Comment by yashraj-dudhe** [comment]
yes whats the error you are getting while installing requirements.txt
and i have assigned you this task
